libraries/ui-strings/src/main/res/values/localazy.xml

```diff
@@ -173,6 +173,14 @@
   <string name="room_timeline_beginning_of_room_no_name">"This is the beginning of this conversation."</string>
   <string name="room_timeline_read_marker_title">"New"</string>
   <string name="screen_analytics_settings_share_data">"Share analytics data"</string>
+  <string name="screen_create_poll_add_option_btn">"Add option"</string>
```

Comment:
@bmarty These should not be here but rather in the poll feature module.
The problem is that the updated filters in the `config.json` exists only in the `julioromano/create_poll_ui` branche at the moment.
What do we do in such situations?

Reply:
That's the issue of having an external tool that does not care about our branch.
I guess for this one it's not a big deal, the string is not used yet (on develop). When you'll merge your PR it will vanish.